templates/div_templates/profile.html

```diff
@@ -2,51 +2,27 @@
-            <form action="/default_avatar" method="post" onsubmit="return submitAjaxForm(this,loadDiv);" enctype="multipart/form-data">
-                <button type="submit" class="changeButton">Use A Random Default Picture</button>
-            </form>
-        </div>
-        <div class="achievements">Achievements</br>
-            <!--All of these are for testing yes it's messy but it's quick-->
-            <img src="./static/avatar1.jpg" class="achievement">
-            <img src="./static/avatar1.jpg" class="achievement">
-            <img src="./static/avatar1.jpg" class="achievement">
-            <img src="./static/avatar1.jpg" class="achievement">
-            <img src="./static/avatar1.jpg" class="achievement">
-            <img src="./static/avatar1.jpg" class="achievement">
-            <img src="./static/avatar1.jpg" class="achievement">
-            <img src="./static/avatar1.jpg" class="achievement">
-            <img src="./static/avatar1.jpg" class="achievement">
-            <img src="./static/avatar1.jpg" class="achievement">
-            <img src="./static/avatar1.jpg" class="achievement">
-            <img src="./static/avatar1.jpg" class="achievement">
-            <img src="./static/avatar1.jpg" class="achievement">
-            <img src="./static/avatar1.jpg" class="achievement">
-            <img src="./static/avatar1.jpg" class="achievement">
-            <img src="./static/avatar1.jpg" class="achievement">
-            <img src="./static/avatar1.jpg" class="achievement">
-            <img src="./static/avatar1.jpg" class="achievement">
-            <img src="./static/avatar1.jpg" class="achievement">
-            <img src="./static/avatar1.jpg" class="achievement">
-            <img src="./static/avatar1.jpg" class="achievement">
-            <img src="./static/avatar1.jpg" class="achievement">
-            <img src="./static/avatar1.jpg" class="achievement">
-            <img src="./static/avatar1.jpg" class="achievement">
-            <img src="./static/avatar1.jpg" class="achievement">
-            <img src="./static/avatar1.jpg" class="achievement">
-        </div>
-    </div>
+<div class = "bookmark">
+<img src = "{{pfp}}" class = "pfpAvatar">
+<div class = "forms">
+<form action="/change_avatar" id="image-form" method="post" onsubmit="return submitAjaxForm(this, loadDiv);" enctype="multipart/form-data">
+    <label for="form-file" class="imageLabel">Image: </label>
+    <input id="pfp-file" type="file" name="upload" class="uploadButton">
+    <button type = "submit" class="changeButton">Change Profile Picture</button>
+    {% if error %}
+    <div class = "error">{{error}}</div>
+    {% endif %}
+</form>
+<form action = "/default_avatar" method = "post" onsubmit="return submitAjaxForm(this,loadDiv);" enctype="multipart/form-data">
+<button type = "submit" class="changeButton">Use A Random Default Picture</button>
+</form>
+</div>
+<div class = "achievements">Achievements</br>
+    {%for i in achievements%}
```

Comment:
Generally, we don't put the spaces around the equals on HTML element attributes, but we do put a space surrounding templating logic, such as `{% for ... %}` and `{{ some_var }}`. There's no hard rule about this, though.